viewing_party/party.py

```diff
@@ -0,0 +1,207 @@
+import math
+
+# wave 1 Creating new movie dictionary 
+
+def create_movie(title, genre, rating):
+    new_movie = {}
+    if not title or not genre or not rating:
+        return None
+    else:
+        new_movie["title"] = title
+        new_movie["genre"] = genre 
+        new_movie["rating"] = rating
+    return new_movie
```

Comment:
I generally recommend creating and initializing the dictionary all at once:

```suggestion
    if not title or not genre or not rating:
        return None
    else:
        new_movie = {
            "title": title,
            "genre": genre ,
            "rating": rating
        }
        return new_movie
```